docs/pages/settings/+Page.mdx

```diff
@@ -50,6 +50,8 @@ Other settings that modify the page HTML shell:
 - [**`lang`**](/lang): Set the page's language (`<html lang>`).
 - [**`htmlAttributes`**](/htmlAttributes): Add `<html>` attributes (e.g. `<html class="dark">`).
 - [**`bodyAttributes`**](/bodyAttributes): Add `<body>` attributes (e.g. `<body class="dark">`).
+- [**`headHtmlBegin`**](/headHtmlBegin): Insert HTML at the beginning of `<head>`.
```

Comment:
Could go in Advanced section? There are very few use cases that aren't already covered by Head / title settings

Reply:
Ok